odoo/cli/module.py

```diff
@@ -172,4 +172,8 @@ def _uninstall(self, parsed_args):
 
     def _force_demo(self, parsed_args):
         with self._create_env_context(parsed_args.db_name) as env:
-            force_demo(env)
+            demo_active = bool(env['ir.module.module'].search_count([('demo', '=', True)]))
+            if demo_active:
+                _logger.info("Demonstration data are already installed...")
+            else:
+                force_demo(env)
```

Comment:
This one changes the behavior of the force demo.
I am not against this change, because the behavior of `force_demo` was not really well defined.
But since it is a feature change, maybe do it in master?

Consider the following use case
1. when installing mdoules, the framework tries to install some demo data, however it fails for some reason. (`module.state == 'installed'`, but `module.demo == False`)
2. then after fixing some problems, we tries to enable all demo modules by force demo when some `module.demo == True`, some `module.demo == False`

and after this change, the cli looks like `install_demo` but not `force_demo`

@lordkrandel what do you think?